lib/src/util.dart

```diff
@@ -6,6 +6,7 @@ library dart_pad.util;
 
 import 'dart:async';
 import 'dart:html';
+import 'package:meta/meta.dart';
```

Comment:
We should add an explicit dep on package:meta in the pubspec - I think we're getting it from a transitive dep right now.

I'd also sort the import directives lexically, and turn on the lint which checks that directives are sorted (`directives_ordering`).

Reply:
Directives ordering is enabled. Which import do you think is out of order?

Reply:
Ah, disregard :) I'm used to seeing a blank line between dart: imports and package: ones, and assumed we didn't have the ordering directive enabled.